src/components/modules/MainPage/LongCard/StyledLongCard.js

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+  width:100rem;
+  height: 17.5rem;
+  height:100%;
+  margin: 0.5%;
+  font-size:1.2rem;
+  border-bottom-right-radius:40px;
+  border-bottom-left-radius:40px;
+  background-color:#C3C0C0;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16), 0 3px 6px rgba(0, 0, 0, 0.23); */
+  display: flex;
+  flex-direction: column;
+  @media (orientation: portrait) {
+    width: 100%;
+    height:100%;
+    margin-bottom: 5%;
+  }
+`;
+
+
+export const HeaderCard = styled.div`
+  position: relative;
+  border-bottom-right-radius:40px;
+  border-bottom-left-radius:40px;
+  width:100rem;
```

Comment:
we try to make the css adaptive to all screens, removing this will allow the card to span to the entire of the screen.

Reply:
remove the width :100rem